### DISCUSSION TITLE
突然发现评论区的表情无法正常显示

### DISCUSSION THREAD

**Discussion by elkan1788** [walinejs/waline#535] [Q&A] [answered]
看讨论区之前有人遇到 过类似的问题，参考 #272 

但按他的说法升级了`vercel`中部署的waline版本后，新发表的评论可以正常显示，但之前的评论仍然是显示为图片地址。

![image](https://user-images.githubusercontent.com/1217096/132431056-76864c43-ca29-43fe-819a-1b2861a6a4fb.png)

在线体验地址： [https://waline.lisenhui.cn/](https://waline.lisenhui.cn/)

求助，有没人解决过类似的问题。

**Comment by lizheming** [comment, marked as answer]
看起来是之前的评论已经转义入库存储了，不太好处理啦~ 建议不重要的就直接忽略吧~
